IFC4x3/PropertyConstants/r/RAISEDRILL_2N81ytBd90BhBPCttgjznX.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="RAISEDRILL_2N81ytBd90BhBPCttgjznX" Name="RAISEDRILL" UniqueId="97201f37-2e72-402e-b2d9-337deab7dc61" />
```

Comment:
Description.

Reply:
Removed according to input from @jontydunn